tools/lambda-promtail/lambda-promtail/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	err := os.Setenv("WRITE_ADDRESS", "http://localhost:8080/loki/api/v1/push")
+	if err != nil {
+		return
+	}
+
+	exitVal := m.Run()
+
+	os.Exit(exitVal)
+}
```

Comment:
I don't think we need/want this test honestly.

Reply:
You are right, I forgot to remove, was playing with this when `init()` function was not playing nice due to missing envs.